src/response/chi0.jl

```diff
@@ -285,6 +285,16 @@ end
         D = mpi_sum(D, basis.comm_kpts)  # equal to minus the total DOS
         δocc_tot = mpi_sum(sum(basis.kweights .* sum.(δocc)), basis.comm_kpts)
         δεF = δocc_tot / D
+        # recompute δocc
+        for ik = 1:Nk
+            for (n, εnk) in enumerate(ε_occ[ik])
```

Comment:
merge loops for less indention: `for ik = ..., (n, εnk) = ...`

Reply:
Agree, done in [89bd331](https://github.com/JuliaMolSim/DFTK.jl/pull/709/commits/89bd3317fe39bcd66af50c1b66b2370fbaba9d8b)